content/testing/fluent-assertions.md

```diff
@@ -0,0 +1,96 @@
+---
+title: FluentAssertions
+showMetadata: true
+editable: true
+showToc: true
+---
+
+สมมติว่าเรามี Data Structure ดังนี้ คำถามคือเราจะ Assert อย่างไร
+
+```c#
+public class Todo
+{
+    public int id  { get; set; }
```

Comment:
Please follow C# naming convention that property is Pascal case.

Reply:
Yes, i’ll